aion/test/test_helper.exs

```diff
@@ -2,3 +2,18 @@ ExUnit.start
 
 Ecto.Adapters.SQL.Sandbox.mode(Aion.Repo, :manual)
 
+defmodule Aion.TestHelpers do
```

Comment:
I guess the conventional way is to put it in the `support/` and name it with `_case`:
![image](https://user-images.githubusercontent.com/15965147/31625785-a24c1bb2-b2a7-11e7-879e-ddcc59b80d46.png)